driver-core/src/main/java/com/datastax/driver/core/policies/WhiteListPolicy.java

```diff
@@ -61,4 +74,51 @@ public boolean apply(Host host) {
       }
     };
   }
+
+  /**
+   * Creates a new policy with the given host names.
+   *
+   * <p>See {@link #fromHosts(LoadBalancingPolicy, String...)} for more details.
+   */
+  public static WhiteListPolicy fromHosts(
```

Comment:
I had considered adding a `WhiteListPolicy(LoadBalancingPolicy childPolicy, Collection<String> hostnames)` but that doesn't work because of type erasure.  Having a static method seems reasonable in any case.

Reply:
I like the static factory method but wouldn't `ofHosts` be more meaningful than `fromHosts`?